app/uk/gov/hmrc/exports/repositories/UpdateSubmissionsTransactionalOps.scala

```diff
@@ -51,17 +53,33 @@ class UpdateSubmissionsTransactionalOps @Inject() (
 
   private lazy val nonTransactionalSession = mongoComponent.client.startSession().toFuture()
 
-  def updateSubmissionAndNotifications(actionId: String, notifications: Seq[ParsedNotification], submission: Submission): Future[Submission] =
+  // All update ops use 'enhancedStatusLastUpdated', loaded at transaction start, for "locking" the document under update.
```

Comment:
Need to update this comment. It should  refer to new 'lastUpdated' field and not mention "locking" as that is not the new approach